src/api/db/migrate/20190704072437_add_column_in_rollout_to_users.rb

```diff
@@ -1,5 +1,5 @@
 class AddColumnInRolloutToUsers < ActiveRecord::Migration[5.2]
   def change
-    add_column :users, :in_rollout, :boolean, default: true, index: true
```

Comment:
So this was ignored, but where is the migration to add this index? I don't see it in this PR.

Reply:
I forgot to add the file with the migration. Added now. Thanks!